### PR TITLE
Print logs and write to logfile at the same time

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Support both printing logs and write to a logfile for the SubprocessLauncher launched processes

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/app_common/launchers/subprocess_launcher.py [hunk 1]
```diff
@@ -47,15 +58,24 @@ def launch_task(self, task_name: str, shareable: Shareable, fl_ctx: FLContext, a
             command = self._script
             env = os.environ.copy()
             command_seq = shlex.split(command)
+            log_file = os.path.join(self._app_dir, f"{task_name}_logfile")
```
Comment (SYangster) [inline nvflare/app_common/launchers/subprocess_launcher.py:61]: If not using `launch_once`, and multiple tasks of the same name are done in succession, then the log file will be overwritten each time. Should we append/have a different log file for each task, or is this ok for now?

Comment (YuanTingHsieh) [reply nvflare/app_common/launchers/subprocess_launcher.py:61]: @SYangster good catch!
I remember this in the beginning but at some point I forgot.

I'll do the append mode.

**Review by SYangster — APPROVED** [review]
Thanks for the fix, LGTM

**Pull request merged**